pkg/synthetictests/event_junits.go

```diff
@@ -47,8 +47,16 @@ func StableSystemEventInvariants(events monitorapi.Intervals, duration time.Dura
 	tests = append(tests, testAPIQuotaEvents(events)...)
 	tests = append(tests, testErrorUpdatingEndpointSlices(events)...)
 	tests = append(tests, testConfigOperatorReadinessProbe(events)...)
-	tests = append(tests, testHttpConnectionLost(events)...)
+	tests = append(tests, testConfigOperatorProbeErrorReadinessProbe(events)...)
```

Comment:
I think you may want to create an alternate factorization if you're going to be creating a lot of these. Perhaps a map from testname to function.